thats_the_way.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+def find_deep_files(path):
+    deep_files = []
+    for file in os.listdir(path):
+        if file.startswith("deep"):
+            deep_files.append(file)
+    return deep_files
```

Comment:
You should aim to write everything in a single line, even if you can. 
This link https://towardsdatascience.com/you-should-stop-craving-for-python-one-liners-5dbf8e0de23d explains it well, even though I disagree with how important memory and runtime are in most python programs.

Reply:
I fixed it